### DISCUSSION TITLE
Problème calcul SPPS

### DISCUSSION THREAD

**Discussion by RemiButtard** [Ifsttar/I-Simpa#305] [Q&A]
Bonjour,
Je suis en train de calculer le TR d'un pièce type ''boite à chaussure''.
Lorsque j'utilise le calcul SPPS sur i-simpa 1.3.4, la plupart de mes valeurs de TR sont ''NaN''. Lorsque je relance le calcul, les valeurs NaN ne sont pas forcément sur les mêmes bandes de fréquence, aléatoirement. J'ai essayé de modifier plusieurs paramètres et je ne vois rien qui agit sur ce problème. Toutes les faces apparaissent dans la fenêtre de visualisation, donc a priori pas de problème à ce niveau.
Quelqu'un a-t-il une explication et une solution ?
Merci d'avance !

**Comment by Picaut** [comment]
"NaN" means "Not A Number" meaning that a calculation can not give a relevant value or that the calculation can not be done.

Assuming the 3D scene is correct, and that a sound source is active, you should check the SPL curves for the corresponding receiver. It is probably because the SPL dynamic is not not enough for the RT calculation. In the SPPS calculation parameters:
- calculate RT30 or RT15 instead of RT60
- and/or increase the number of sound particles “Number of sound particles per source”
- and/or increase the duration “Simulation length (s)”

**Reply by RemiButtard** [reply]
D'accord, merci beaucoup, effectivement j'arrive maintenant à avoir des valeurs, mais ces valeurs sont très grandes (3 ou 4 secondes avec SPPS contre 0.46s avec la méthode de Sabine). J'ai beau jouer sur les différents paramètres de simulation je n'arrive pas à me rapprocher d'un TR un peu réaliste.

**Reply by Picaut** [reply]
Sabine and SPPS may be different for example if the room is elongated, or/and the absorption is important and not homogenously distributed on the room surfaces. If you could share your project or images ou Schroeder curves, may be I could give your additional comments.
